core/templates/dev/head/pages/exploration_editor/simple_editor_tab/simple_editor_sidebar_directive.html

```diff
@@ -112,6 +112,9 @@
   simple-editor-sidebar .highlighted {
     background-color: #c5e1a5;
   }
+  simple-editor-sidebar .error {
```

Comment:
OK, glad to hear that you figured out how to use the material icons!

I'll defer to @andromfins on your other question, as I don't think I have enough context to answer it, apart from the general principle I mentioned a couple of comments before. But the fact that you would need to write additional functions in order to implement the full functionality doesn't affect that comment.